libsel4simple-default/src/arch/riscv/default.c

```diff
@@ -19,10 +19,17 @@ seL4_Error simple_default_get_irq(void *data, int irq, seL4_CNode root, seL4_Wor
     return seL4_IRQControl_Get(seL4_CapIRQControl, irq, root, index, depth);
 }
 
+seL4_Error simple_default_get_irq_trigger(void *data, int irq, int trigger, UNUSED int core, seL4_CNode root,
+                                          seL4_Word index, uint8_t depth)
+{
+    return seL4_IRQControl_GetTrigger(seL4_CapIRQControl, irq, trigger, root, index, depth);
```

Comment:
Should this be using `seL4_IRQControl_GetTriggerCore` like on ARM? https://github.com/seL4/seL4_libs/blob/312048073fdbedafbd9dbafaf01f8e39bbc0f141/libsel4simple-default/src/arch/arm/default.c#L27

Reply:
Nevermind, that invocation does not exist on RISC-V.